include/dlaf/matrix/print_human.h

```diff
@@ -0,0 +1,69 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+#include "dlaf/common/index2d.h"
+#include "dlaf/common/range2d.h"
+#include "dlaf/matrix/matrix.h"
+#include "dlaf/matrix/tile.h"
+#include "dlaf/types.h"
+
+namespace dlaf {
+
+namespace format {
+struct human {};
+}
+
+namespace matrix {
+
+namespace internal {
+
+/// Print a matrix in a human readable format to standard output
+// TODO: now is printed to standard output, would it be better to save it into a file?
+template <class T>
+void print(format::human, Matrix<T, Device::CPU>& mat) {
+  SizeType nrow = mat.size().rows();
```

Comment:
As a distributed matrix will produce incorrect output, if the matrix is distributed just print the basic matrix info.